overlord/ifacestate/handlers.go

```diff
@@ -303,3 +306,24 @@ func (m *InterfaceManager) doDisconnect(task *state.Task, _ *tomb.Tomb) error {
 	setConns(st, conns)
 	return nil
 }
+
+func (m *InterfaceManager) doDiscardNamespace(task *state.Task, _ *tomb.Tomb) error {
```

Comment:
Interfaces can tweak it, but this sounds like a property of every snap. We have unlink-snap, unlink-current-snap, clear-snap, and discard-snap. Why do we need another task?